src/TLuaInterpreter.h

```diff
@@ -107,7 +107,7 @@ class TLuaInterpreter : public QThread
     void adjustCaptureGroups(int x, int a);
     void clearCaptureGroups();
     bool callEventHandler(const QString& function, const TEvent& pE, const QEvent* qE = nullptr);
-    static QString dirToString(lua_State*, int);
+    static QString dirToString(lua_State*, int, bool isForCustomExits = false);
```

Comment:
*Well* this function is a helper currently for two cases the `setExitWeight` and `addCustomExit` functions and that third argument changes the case for the return values so that the default value is the lower case needed for the SetExitWeight command but gets switched to the upper case needed *for Custom Exits* when a `true` is supplied - so the wording did actually make sense to *me*. :stuck_out_tongue_closed_eyes:

Reply:
Third argument no more is... :rofl: